enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -99,259 +116,211 @@ This is often required for industries such as the US public sector, healthcare o
 
 ### Implementation Details
 
-#### Assumptions
-
-* A pipeline shall support at least one output.
-* A pipeline may support multiple outputs.
-* Outputs that require secrets which are not created by the cluster-logging-operator shall be created and managed by the administrator of the endpoint
-* Secrets shall have keys of: `tls.crt`, `tls.key`, `ca-bundler.crt` which point to the respective certificates for which they represent. Secrets shall have the key `shared_key` for use when using `forward` in a secure manner
-* The operator will only support a single instance of log forwarding named `instance` for at least the tech preview release
-* Log forwarding is enabled for tech preview when the instance of `clusterloggging` is annotated with: `clusterlogging.openshift.io/logforwardingtechpreview: enabled`
+* A pipeline associates multiple input names with multiple output names.
+* Users cannot define new inputs (for initial release)
+* Users can define new outputs, output configuration includes:
+  - `type` (e.g. `Syslog`, `Fluent`)
+  - `destination` URL for the endpoint.
+  - `secretRef` referring to a secret object used for secure connections.
+    - secrets must be in the `openshift-logging` namespace (for GA)
+    - a secretRef with no namespace defaults to the `openshift-logging` namespace.
```

Comment:
is this line necessary due to the line above stating they must be in a specific namespace?

Reply:
Yes, maybe I need to rephrase: the secrets (in first release anyway) must be in namespace osl.
The SecretReferences is has { name: xxx, namespace: yyy } What I'm trying to say is you can just leave out the namespace: in in your YAML, if it's  missing we'll assume the intended one.

Reply:
We could just have a secret name in the YAML, and assume that secrets will always be in the osl namespace - but I'd rather not tie us to that. When we bring in namespace-scoped forwarding or other types of forwarding, it may no longer be OK to assume there's a single namespace for secrets.

Reply:
How about this: